src/config/fuse-config.js

```diff
@@ -1,5 +1,6 @@
 // Configuration for Fuse.js (Fuzzy Search)
 
+// TODO: Use column-generator
```

Comment:
Need to import column-generator here, for availableIncidentTableColumns, availableAlertTableColumns,